src/view-components/dashboard/dashboard.component.tsx

```diff
@@ -27,34 +28,50 @@ export default function Dashboard(props: DashboardProps) {
   }
 
   return (
-    <div className={styles.container}>
-      <div
-        className={styles.dashboard}
-        style={{ gridTemplateColumns: getColumnsLayoutStyle() || 2 }}
-      >
-        {widgets.map((widget, index) => {
-          const widgetConfig = props.dashboardConfig.widgets[index];
-          let rows = (widgetConfig.layout && widgetConfig.layout.rowSpan) || 1;
-          let columns =
-            widgetConfig.layout && (widgetConfig.layout.columnSpan || 1);
-          return (
-            <div
-              key={widgetConfig.name}
-              className={styles.widgetContainer}
-              style={{
-                gridRow: `span ${rows}`,
-                gridColumn: `span ${columns}`
-              }}
-            >
-              {widget}
-            </div>
-          );
-        })}
+    <>
+      <div className={styles.container}>
+        <div className={styles.dashboard} style={{ gridTemplateColumns: 1 }}>
+          <ExtensionSlot
+            extensionSlotName={`patient-chart-${sanitize(
```

Comment:
I'm not sure sanatize makes sense here. After all, this name has to match what was given in the widgets. If a user enters something wrong in the config it should not show up, or?

Reply:
This code should go away in the not-too-distant future, once we redo patient-chart to be fully extension-based.